### PR TITLE
Feature: Target a specific number

### PR DESCRIPTION
**Pull request by Rits1272**
## Issue that this pull request solves

Closes: #133 

## Proposed changes
An option to target a specific number, without the need to include in your phonebook

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots
If the user will type a number in target then the browser will automatically open : https://wa.me/target_number
Screenshot (50)
Screenshot (51)

### CODE REVIEW

**Comment by Rits1272** [comment]
Please review @rpotter12 @xandao6

File: wplay/scheduleMessage.py [hunk 1]
```diff
@@ -24,5 +25,6 @@
         sys.exit()
     message : list[str] = io.ask_user_for_message_breakline_mode()
     print("Your message is scheduled at : ", time_)
+
```
Comment (rpotter12) [inline wplay/scheduleMessage.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5246692)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5246692&bid=17195303)

**Comment by xandao6** [comment]
Simple and Nice!

**Comment by xandao6** [comment]
It doesn't solve the problem. We still need to click in send message...

**Pull request merged**